README.md

```diff
@@ -127,6 +127,16 @@ class ExampleTable extends Table
 }
 ```
 
+#### Empty Values
+By default, `['', false, null]` are treated as empty values and will be filtered out. If you wish to
+alter this behavior, you can overwrite the values using `emptyValues` key.
+
+```php
+$this->addBehavior('Search.Search', [
+    'emptyValues' => ['']
```

Comment:
Does this actually work? Or do we need explicit shallow merging here to overwrite because CakePHP uses the wrong merge operation by default?

Reply:
This will work. When emptyValues is supplied when initializing, the shallow merge is used to overwrite emptyValues

Reply:
Reference: https://github.com/dilab/search/blob/master/tests/TestCase/Model/Behavior/SearchBehaviorTest.php#L206

Reply:
@dereuromark Check the behavior's `initialize()` method. I ensured he made required change so that `emptyValues` key is overwritten not merged :slightly_smiling_face: